client/webserver/site/src/js/markets.js

```diff
@@ -1669,10 +1676,11 @@ export default class MarketsPage extends BasePage {
   /* lotChanged is attached to the keyup and change events of the lots input. */
   lotChanged () {
     const page = this.page
-    const lots = parseInt(page.lotField.value)
+    const lots = parseInt(page.lotField.value) || 0
     if (lots <= 0) {
-      page.lotField.value = 0
+      page.lotField.value = ''
```

Comment:
I actually prefer to insert the zero in this case, since it helps indicate the problem with the user's input.